backend/src/main/kotlin/com/tngtech/apicenter/backend/config/SecurityConfiguration.kt

```diff
@@ -25,12 +26,33 @@ class SecurityConfiguration : WebSecurityConfigurerAdapter() {
             .csrf().disable()
             .authorizeRequests()
             .antMatchers("/sessions").permitAll()
-            .and()
-            .authorizeRequests()
+
+            .antMatchers("/").permitAll()
+            .antMatchers("/3rdpartylicenses.txt").permitAll()
+            .antMatchers("/favicon.ico").permitAll()
+            .antMatchers("/index.html").permitAll()
+            .antMatchers("/main.js").permitAll()
+                .antMatchers("/main.js.map").permitAll()
```

Comment:
Could you please fix the indentation?

Reply:
Heh, there was me thinking this made more sense 😉 Sure.